test/sanitytest.c

```diff
@@ -130,22 +134,72 @@ static int test_sanity_memcmp(void)
     return CRYPTO_memcmp("ab", "cd", 2);
 }
 
-static int test_sanity_sleep(void)
+static const struct sleep_test_vector {
+    uint64_t val;
+} sleep_test_vectors[] = { { 0 }, { 1 }, { 999 }, { 1000 }, { 1999 } };
```

Comment:
Hmm. This makes the test duration 4 seconds long (an increase of 3 seconds). That's really quite expensive. Can't we use smaller numbers here?